src/output/naive.rs

```diff
@@ -22,16 +22,21 @@ use std::sync::Mutex;
 use timely;
 use timely::dataflow::operators::*;
 
-pub(super) fn compute(dump_enabled: bool, all_facts: AllFacts) -> Output {
+pub(super) fn compute(dump_enabled: bool, all_facts: AllFacts, workers: u32) -> Output {
     let result = Arc::new(Mutex::new(Output::new(dump_enabled)));
 
     // Use a channel to send `all_facts` to one worker (and only one)
     let (tx, rx) = mpsc::channel();
     tx.send(all_facts).unwrap();
     mem::drop(tx);
     let rx = Mutex::new(rx);
+    let mut dataflow_arg = Vec::new();
+    if workers > 1 {
+        let item = format!("{}{}", "w", workers);
+        dataflow_arg.push(item);
```

Comment:
I think this should be

```
dataflow_arg.push(format!("-w"));
dataflow_arg.push(format!("{}", workers));
```

though one thing may work too

Reply:
done!